queue/example_test.go

```diff
@@ -16,6 +16,7 @@ import (
 	"github.com/tarantool/go-tarantool/queue"
 )
 
+//nolint
```

Comment:
I would fix it instead of suppressing:

```diff
-//nolint
 // Example demonstrates an operations like Put and Take with queue.
 func Example_simpleQueue() {
        cfg := queue.Cfg{
@@ -74,7 +73,7 @@ func Example_simpleQueue() {
                return
        }
 
-       task, err = q.TakeTimeout(2 * time.Second)
+       task, _ = q.TakeTimeout(2 * time.Second)
        if task != nil {
                fmt.Printf("Task should be nil, but %d", task.Id())
                return
```

Reply:
Fixed, seems better to add the check of error.